addons/website_forum/static/src/js/website_forum.js

```diff
@@ -317,13 +320,19 @@ publicWidget.registry.websiteForum = publicWidget.Widget.extend({
             } else if (data.success) {
                 var elem = $link;
                 if (data.success === 'post_flagged_moderator') {
-                    elem.data('href') && elem.html(' Flagged');
-                    var c = parseInt($('#count_flagged_posts').html(), 10);
-                    c++;
-                    $('#count_flagged_posts').html(c);
+                    let $child = elem.children(),
+                        $count_flagged_posts = $('span#count_flagged_posts'),
```

Comment:
```suggestion
                        $count_flagged_posts = this.$('span#count_flagged_posts'),
```
or `self` I let you check

Reply:
And the `span` part you added is probably useless no? it's an ID anyway, there shouldn't be more than one element..